workflow/rocoto_funcs/upp.py

```diff
@@ -13,8 +13,8 @@ def upp(xmlFile, expdir, do_ensemble=False):
     cycledefs='prod'
     # metatask (nested or not)
     fhr=os.getenv('FCST_LENGTH','9')
-    #meta_hr=''.join(f'{i:03d} ' for i in range(int(fhr)+1)).strip()
-    meta_hr=''.join(f'{i:03d} ' for i in range(int(fhr)+1)).strip()[4:] #remove '000 ' as no f000 diag and history files for restart cycles yet, gge.debug
+    meta_hr=''.join(f'{i:03d} ' for i in range(int(fhr)+1)).strip()
+    #meta_hr=''.join(f'{i:03d} ' for i in range(int(fhr)+1)).strip()[4:] #remove '000 ' as no f000 diag and history files for restart cycles yet, gge.debug
```

Comment:
If we can do mpassit and upp at f000 now, let's remove line 17. 
But let's double check whether we can do them at f000.

Reply:
We can do f000 mpassit and upp now.

Reply:
> We can do f000 mpassit and upp now.

This is great! We can remove line 17 now. Thanks!